src/Features/CSharp/Portable/Structure/Providers/DestructorDeclarationStructureProvider.cs

```diff
@@ -32,6 +32,7 @@ internal class DestructorDeclarationStructureProvider : AbstractSyntaxNodeStruct
             spans.AddIfNotNull(CSharpStructureHelpers.CreateBlockSpan(
                 destructorDeclaration,
                 destructorDeclaration.ParameterList.GetLastToken(includeZeroWidth: true),
+                compressEmptyLines: false,
```

Comment:
but what if i have two destructors! :D

on a serious note, why no `!...IsNode()` check?

Reply:
Easier to just not have any special handling for this rare node.

Reply:
👍